ui/css/treeherder-navbar.css

```diff
@@ -0,0 +1,305 @@
+/*
+ * Main navbar settings
+ */
+
+#global-navbar-container {
+    background-color: #273038;
+    margin-bottom: 0px;
+    flex: none;
+    -webkit-flex: none;
+}
+
+#th-global-navbar-top {
+    padding-left: 0;
+    border-bottom: 1px solid black;
+}
+
+.navbar {
+    min-height: 36px;
+    margin-bottom: 0px;
+}
+
+.navbar .dropdown-menu {
+    max-height: 600px;
+    overflow: auto;
+    margin-top: 10px;
+    padding-bottom: 8px;
+}
+
+#th-logo {
+    padding-left: 14px;
+    border-left: 0;
+}
+
+.btn-right-navbar {
+   font-size: 13px;
+}
+
+/*
+ * Right hand upper navbar
+ */
+
+.navbar-right {
+    /* Override boostrap 3.3.5 which doesn't work with the way we align our navbar */
+    margin-right: 0px;
+}
+
+.navbar-collapse {
+    /* Override bootstrap for flush right navbar menus */
+    padding-right: 0;
+}
+
+.nav-menu-btn {
+    margin-right: -4px;
+    padding-left: 14px;
+    padding-right: 14px;
+}
+
+.nav-help-btn {
+    margin-right: -4px;
+    padding-left: 12px;
+    padding-right: 8px;
+}
+
+.nav-help-icon {
+    font-size: 18px;
+    margin-right: 2px;
+}
+
+.nav-user-icon {
+    display: inline-block;
+    margin-right: 4px;
+    padding: 3px 4px;
+    vertical-align: text-bottom;
+    border-radius: 1px;
+    background: linear-gradient(#44adf3, #287cc2);
+}
+
+.nav-user-icon span:first-child {
+    margin: 0 auto;
+    font-size: 12px;
+    color: #fff;
+}
+
+.nav-login-btn {
+    padding-left: 22px;
+    padding-right: 27px;
+}
+
+.nav-persona-btn {
+    padding-left: 14px;
+    padding-right: 14px;
+}
+
+.th-context-navbar {
+    background-color: #354048;
+    overflow: visible;
+}
+
+.nav-panel-help-text {
+    color: #2b393f;
+    margin-bottom: 10px;
+}
+
+/*
+ * Left hand lower navbar
+ */
+
+.watched-repo-main-btn {
+    border-right: 0;
+    padding-right: 5px;
+}
+
+.watched-repo-info-btn {
+    border-right: 0;
+}
+
+.watched-repo-dropdown-item {
+    margin: 0px 10px;
+}
+
+.watched-repo-dropdown-item > a {
+    white-space: normal;
+}
+
+.watched-repo-navbar {
+    overflow: visible;
+}
+
+.tree-closed {
+    color: rgb(161, 52, 53);
+}
+
+.tree-open {
+    color: green;
+}
+
+.tree-approval {
+    color: #fb9910;
+}
+
+.tree-unavailable {
+    color: lightgray;
+}
+
+th-watched-repo {
+    /* Required for Chrome */
+    float: left;
```

Comment:
We didn't appear to need a `display: block` here so I removed it during the transfer. I tested and the fix is still required for Chrome. Either this, or a pull left in the markup I suppose.